Java language level for Scene Builder Kit

<!--- Provide a brief summary of the issue -->
Kit and App are built with same Java level as JavaFX release (currently 23).
While this is not an issue for App, which is distributed with its own JRE, if other third party projects want to use Kit, they are forced to set up the same level, and this shouldn't be the case: Kit should be built with the lower level possible (currently 21) 

### Expected Behavior
<!--- Tell us what should happen -->
Kit should work for any Java 21+ project.

### Current Behavior
Adding Kit to a project running with Java level 21 gives:
```
[ERROR] [4,38] module not found: com.gluonhq.scenebuilder.kit
```

--- a/README.md
+++ b/README.md
@@ -35,7 +35,7 @@ Gluon Scene Builder is frequently released, and this is only required in case yo
 
 These are the requisites:
 
-* A recent version of [JDK 11 or later](https://www.oracle.com/technetwork/java/javase/downloads/index.html) for building 'master' branch
+* A recent version of [JDK 21 or later](https://www.oracle.com/technetwork/java/javase/downloads/index.html) for building 'master' branch
 * A recent version of [JDK 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) for building '8u-dev' branch
 
 ### How to build Scene Builder ###

--- a/pom.xml
+++ b/pom.xml
@@ -16,7 +16,7 @@
     </modules>
 
     <properties>
-        <java.version>23</java.version>
+        <java.version>21</java.version>
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <maven.compiler.target>${java.version}</maven.compiler.target>
         <maven.compiler.release>${java.version}</maven.compiler.release>

